src/Sylius/Behat/Behaviour/Toggles.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Behaviour;
+
+use Behat\Mink\Element\NodeElement;
+use Sylius\Behat\Page\ElementNotFoundException;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+trait Toggles
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function enable()
+    {
+        $toggleableElement = $this->getToggleableElement();
+        $this->assertCheckboxIsNotCheckedYet($toggleableElement, false);
+
+        $toggleableElement->check();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function disable()
+    {
+        $toggleableElement = $this->getToggleableElement();
+        $this->assertCheckboxIsNotCheckedYet($toggleableElement, true);
+
+        $toggleableElement->uncheck();
+    }
+
+    /**
+     * @param NodeElement $toggleableElement
+     * @param bool $expectedState
+     *
+     * @throws \RuntimeException
+     */
+    private function assertCheckboxIsNotCheckedYet(NodeElement $toggleableElement, $expectedState)
```

Comment:
Maybe ``assertCheckboxState``? ``assertCheckboxIsNotCheckedYet`` seems like it checks only not-checked checkbox, but it can checks checked as well.